v3/src/components/case-table/case-table-tool-shelf-button.tsx

```diff
@@ -101,8 +100,11 @@ const CaseTableToolShelfMenuList = observer(
               onClick={()=>createOrShowTableOrCardForDataset(dataset)} data-testid={`tool-shelf-table-${tileTitle}`}>
               <TableIcon className="menu-icon case-table-icon"/>
               {tileTitle}
-              <TrashIcon className="menu-icon tool-shelf-menu-trash-icon"
-                  onClick={() => handleOpenRemoveDataSetModal(dataset.dataSet.id)} />
+              <Button className="menu-list-button tool-shelf-menu-trash">
```

Comment:
This class name change broke a cypress test it is looking for `tool-shelf-menu-trash-icon` probably best to just update the cypress test.

Reply:
Fixed